docs/additional-resources/videos.table.js

```diff
@@ -4,59 +4,187 @@ import React from "react";
 // Please respect chronological (date) order when adding new entries.
 // ----------------------------------------------------------------------------
 export const videos = [
+  {
+    title: "Overcoming the ‘Testing Sucks’ Mentality with Pester & ChatGPT",
+    author: "Jaap Brasser",
+    date: "2023-06-22",
+    url: "https://www.youtube.com/watch?v=5HhTVeV5d5k",
+    event: "PowerShell Conference EU 2023",
+  },
+  {
+    title: "The rollercoaster of upgrading dbachecks to Pester v5 and Sampler and teamwork",
+    author: "Jess Pomfret, Rob Sewell",
+    date: "2023-06-21",
+    url: "https://www.youtube.com/watch?v=Ff8YCogXpU0",
+    event: "PowerShell Conference EU 2023",
+  },
+  {
+    title: "Test Your Code Quality with Pester and PSScriptAnalyzer",
+    author: "Leo Visser",
+    date: "2023-06-20",
+    url: "https://www.youtube.com/watch?v=zO3-8xgTar8",
+    event: "PowerShell Conference EU 2023",
+  },
```

Comment:
Move to versioned_v4 docs?

Reply:
AFAICT this video from Leo Visser uses Pester V5.

Reply:
Yeah that was a mistake. Intended to choose pre-2020 links. Good catch :)